set_hugepages.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
do we need this?

Reply:
It's required as, it's not present in ipdk diectory.